### PR TITLE
feat: fail on non 200 response

### PR DESCRIPTION
**Pull request by RadekMolenda**
By default the response not returning 200 should exit test script with failure

### CODE REVIEW

**Comment by tanguyantoine** [comment]
Tests maybe :-)

**Comment by RadekMolenda** [comment]
@tanguyantoine updated

File: invoke/test/entrypoint.bats [hunk 1]
```diff
@@ -9,6 +9,7 @@ function setup() {
 }
 
 @test "raises if BEARER_API_KEY is missing" {
+  echo $GITHUB_WORKSPACE/invoke/entrypoint.sh
```
Comment (tanguyantoine) [inline invoke/test/entrypoint.bats]: do we need this line?

File: invoke/test/entrypoint.bats [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bats
 
-PATH="$PATH:$BATS_TEST_DIRNAME/bin"
+PATH="$BATS_TEST_DIRNAME/bin:$PATH"
```
Comment (tanguyantoine) [inline invoke/test/entrypoint.bats]: 👏

File: invoke/test/entrypoint.bats (hunk 1 above)
Comment (RadekMolenda) [inline invoke/test/entrypoint.bats]:
```suggestion
```

Comment (RadekMolenda) [reply invoke/test/entrypoint.bats]: Removed

Comment (tanguyantoine) [reply invoke/test/entrypoint.bats]: 🤦‍♂

**Comment by RadekMolenda** [comment]
@tanguyantoine bump

**Pull request merged**